### PR TITLE
Fix/66685 wmpl fixes

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/66685

This PR adds a WPML integration utility class and fixes an issue that would yield notices in edge cases when activating/deactivating WPML with TEC/PRO installed.

### CODE REVIEW

File: src/Tribe/Integrations/WPML/Utils.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	 *
+	 * @param array $strings
+	 *
+	 * @return array
+	 */
+	public static function get_wpml_i18n_strings( array $strings ) {
+		$tec = Tribe__Events__Main::instance();
+
+		$domains = apply_filters(
+			'tribe_events_rewrite_i18n_domains', array(
```
Comment (bordoni) [inline src/Tribe/Integrations/WPML/Utils.php]: The spading here is odd

Comment (lucatume) [inline src/Tribe/Integrations/WPML/Utils.php]: Latest commit should fix it

Comment (lucatume) [inline src/Tribe/Integrations/WPML/Utils.php]: ♠️

**Comment by bordoni** [comment]
All good now.

**Comment by lucatume** [comment]
Thanks!

**Pull request merged**